core/consensus/authority/impl/schedule_node.hpp

```diff
@@ -24,13 +24,24 @@ namespace kagome::authority {
     ScheduleNode(const std::shared_ptr<const ScheduleNode> &ancestor,
                  primitives::BlockInfo block);
 
+    /// Creates schedule node as root
+    /// @result schedule node
     static std::shared_ptr<ScheduleNode> createAsRoot(
         primitives::BlockInfo block);
 
+    /// Ensure if all significant changes is applied and node is ready to
+    /// schedule next change
+    /// @result success or error
     outcome::result<void> ensureReadyToSchedule() const;
 
+    /// Changes actual authorities as if corresponding block is finalized or not
+    /// @param finalized - true if block is finalized
     void adjust(bool finalized);
```

Comment:
```suggestion
    void adjust(IsBlockFinalized finalized);
```
shouldn't it be?

Reply:
Also, is it by default for finalized block or not? It can be just `adjustForFinalized` and called only for finalized blocks, for example.

Reply:
Finalization state makes sense only for 'scheduled changes' and 'on resume', and does not for 'forced changes' and 'on pause'.